openslides/core/views.py

```diff
@@ -24,6 +25,14 @@
 from .serializers import CustomSlideSerializer, TagSerializer
 
 
+class AngularView(utils_views.View):
```

Comment:
The angular template language and the django template language are to similar at that we can send the angular base template though the django template language.

Also, I like the possibility to configure the webserver (nginx, apache) to serve the angular base template. For that reason it lies in static so it can be copied with "collect static" to a place where nginx can handle it.

Also, all other angular templates are in the static folder. As template developer you only have to define with a higher priority to static to override all templates.

I think we can live with 5 lines of extra code instead of 2 lines for a normal template view.